<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: black;
background-color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>611</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: orange;</string>
       </property>
       <property name="text">
        <string>Menu</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="menu_table">
       <property name="styleSheet">
        <string notr="true">QTableWidget {
    border: 2px solid white;
    border-radius: 5px;
	color: black;
}

QTableWidget::item {
    padding: 5px;
}

QHeaderView::section {
    background-color: orange;
    color: white;
    padding: 4px;
}

QTableWidget::item:selected {
    background-color: orange;
    color: white;
}</string>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="autoScroll">
        <bool>false</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollPerItem</enum>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Ratings</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Num Ratings</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Chef ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Description</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>80</y>
      <width>160</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLineEdit" name="id_num">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid orange; border-radius: 10px;
background-color:white;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Enter Worker ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="error_label">
       <property name="styleSheet">
        <string notr="true">color:rgb(162, 0, 0)</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remove_btn">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: orange;
	color: white;
	border: 10px solid orange;
	border-radius: 12px;
}

QPushButtonHover {
	background-color: rgb(57, 57, 57);
}</string>
       </property>
       <property name="text">
        <string>Remove Item</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add_btn">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: orange;
	color: white;
	border: 10px solid orange;
	border-radius: 12px;
}

QPushButtonHover {
	background-color: rgb(57, 57, 57);
}</string>
       </property>
       <property name="text">
        <string>Add Food</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="file_complain">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: orange;
	color: white;
	border: 10px solid orange;
	border-radius: 12px;
}

QPushButtonHover {
	background-color: rgb(57, 57, 57);
}</string>
       </property>
       <property name="text">
        <string>File Importer Complain</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="refresh">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: orange;
	color: white;
	border: 10px solid orange;
	border-radius: 12px;
}

QPushButtonHover {
	background-color: rgb(57, 57, 57);
}</string>
       </property>
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
